labgrid/driver/gpiodriver.py

```diff
@@ -0,0 +1,46 @@
+"""All GPIO-related drivers"""
+import attr
+
+from ..factory import target_factory
+from ..protocol import DigitalOutputProtocol
+from ..resource.base import SysfsGPIO
+from ..resource.remote import NetworkSysfsGPIO
+from ..step import step
+from .common import Driver
+from ..util.agentwrapper import AgentWrapper
+
+
+@target_factory.reg_driver
+@attr.s(cmp=False)
+class GpioDigitalOutputDriver(Driver, DigitalOutputProtocol):
+
+    bindings = {
+        "gpio_provider": {SysfsGPIO, NetworkSysfsGPIO},
```

Comment:
```suggestion
        "gpio": {SysfsGPIO, NetworkSysfsGPIO},
```
_provider doesn't add information here.